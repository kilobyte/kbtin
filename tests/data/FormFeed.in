#chr ff 12
#showme bare${ff}fin
#sys {printf 'bare\014fin\n'}
#sys {printf 'cls0\033[0Jfin\n'}
#sys {printf 'cls1\033[1Jfin\n'}
#sys {printf 'cls2\033[2Jfin\n'}
#rem ]]] (dumb m4)
