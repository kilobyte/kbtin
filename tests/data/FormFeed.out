barefin
barefin
cls0fin
cls1fin
cls2fin
